vllm/executor/ray_gpu_executor.py

```diff
@@ -106,12 +106,17 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
         # The remaining workers are the actual ray actors.
         self.workers: List[RayWorkerWrapper] = []
 
+        # Used in ray compiled DAG: indexed first by PP rank, and then TP rank.
+        self.pp_tp_workers: List[List[RayWorkerWrapper]] = []
```

Comment:
Comment the first dimension is for TP and the second dimension is for PP?

Reply:
Added additional comment.